internal/setup/setup_test.go

```diff
@@ -1562,44 +1562,42 @@ var setupTests = []setupTest{{
 		`,
 	},
 	relerror: `slice mypkg_myslice path /path/\*\* has invalid generate options`,
+}, {
+	summary: "chisel-v1 is deprecated",
+	input: map[string]string{
+		"chisel.yaml": `
+			format: chisel-v1
+			archives:
+				foo:
+					version: 22.04
+					components: [main, universe]
+					suites: [jammy]
+					v1-public-keys: [test-key]
+					default: true
+			v1-public-keys:
+				test-key:
+					id: ` + testKey.ID + `
+					armor: |` + "\n" + testutil.PrefixEachLine(testKey.PubKeyArmor, "\t\t\t\t\t\t") + `
+		`,
+	},
+	relerror: `chisel.yaml: unknown format "chisel-v1"`,
```

Comment:
[Question for reviewer]: The only question is whether we should introduce a better error message for `chisel-v1` that says something like "consider an update if available" just like we do for invalid `generate: ` values.

Reply:
Per the conversation we had, to introduce such a message we need to do that at the very top layer, in the user interface itself (CLI, or GUI) as otherwise we cannot tell the context in which the library/package is being used. Even there, sometimes it may be misleading as the tool may be leveraged in servers and the error reported in a way that is impossible for the receiver of the message to act on the suggestion. For that kind of reason, it is usually better to report the error with a good error message which allows the reader to grasp what is going on by themselves, which seems like the case here already.